src/test/regress/citus_tests/config.py

```diff
@@ -318,6 +318,12 @@ def __init__(self, arguments):
         self.new_settings = {"citus.shard_count": 1}
 
 
+class CitusUseCitusManagedTablesClusterConfig(CitusBaseClusterConfig):
```

Comment:
Apart from Onder's comment, when adding a new config, we should extend either CitusDefaultClusterConfig or CitusMXBaseClusterConfig, otherwise the config won't be run:

https://github.com/citusdata/citus/blob/master/src/test/regress/citus_tests/config.py#L59-L66

https://github.com/citusdata/citus/tree/master/src/test/regress/citus_tests/arbitrary_configs#adding-a-new-config

Reply:
We should probably only have a single base class, `CitusDefaultClusterConfig ` should cover MX as well